bluesky/plan_stubs.py

```diff
@@ -650,15 +652,21 @@ def kickoff(obj, *, group=None, wait=False, **kwargs):
     :func:`bluesky.plan_stubs.collect`
     :func:`bluesky.plan_stubs.wait`
     """
-    ret = (yield Msg('kickoff', obj, group=group, **kwargs))
-    if wait:
-        yield from _wait(group=group)
-    return ret
+    objs = [obj]+[check_supports(arg, Flyable) for arg in args]
+
+    statuses: Dict[Flyable, Status] = {}
+    for obj in objs:
+        ret = yield Msg('kickoff', obj, group=group, **kwargs)
+        statuses[obj] = ret
+        if wait:
+            yield from _wait(group=group)
```

Comment:
```suggestion
    if wait:
        yield from _wait(group=group)
```
Only wait once when we have kicked them all off